TSOClient/FSO.Server.Database/DatabaseScripts/changes/0032_fso_mentors.sql

```diff
@@ -0,0 +1,5 @@
+CREATE TABLE IF NOT EXISTS `fso_mentors` (
```

Comment:
What's the line of thinking of making this a separate table, rather than attaching it to the Users table? Are you thinking that more mentor related fields might be added to this table?

Reply:
Two things with that, number one was that I didn't necessarily know if you wanted the Users table to be appended with something like this at the time, knowing this now, I am agree this should be in the Users table. Number two is exactly that, I do think that badges could have a practical use in the future where a user can have a badge for something besides Mod/Admin, and Mentor. (Contributer?)

Reply:
In that case, why not make it a "badges" table, with "badge_type" as a field? One avatar can have multiple badges - though for it to make the most general sense it might have to be linked on `avatar_id` rather than `user_id`, if you wanted to add badges for individual sims.

Reply:
I could see such a system being useful in future if we decide to award badges for an event, and then show them on a badge tab in the Person Page.